doc/commands.doc

```diff
@@ -1452,34 +1452,58 @@ contains \c TEST, or \c DEV
   | \%A | The full weekday name.
   | \%lA | As \%A but when not by default  the case and when possible the first character is converted to uppercase.
   | &nbsp; | &nbsp;
+  | \%H |The hour (0-23).
+  | \%0H | The hour (0-23). If the result is a single digit, it is prefixed with 0.
+  | \%I | The hour (0-11).
+  | \%0I | The hour (0-11). If the result is a single digit, it is prefixed with 0.
+  | \%M | The minute.
+  | \%0M | The minute. If the result is a single digit, it is prefixed with 0.
+  | \%S | The second.
+  | \%0S | The second. If the result is a single digit, it is prefixed with 0.
+  | &nbsp; | &nbsp;
   | \%\% | A \% character.
 
   Note that the \<format\> has to be between double quotes.
 
-  In case the \<date\> is specified it to has to have the following representation:
-  - one or 2 digits for the day
-  - a minus sign
-  - one or 2 digits for the month
-  - a minus sign
-  - 4 digits for the year
+  In case the \<date_time\> is specified it has to have the following representation:
+  - optional `date` where `date` is:
+    - one or 2 digits for the day
+    - a minus sign
+    - one or 2 digits for the month
+    - a minus sign
+    - 4 digits for the year
+  - optional `time` where `time` is:
+    - whitespace
+    - one or 2 digits for the hours
+    - a colon sign
+    - one or 2 digits for the minutes
+    - optionally
+      - a colon sign
+      - 2 digits for the seconds
   .
-  in case the \<date\> is not specified the current date is used.
+  in case the \<date_time\> is not specified the current date and time are used.
 
   Here is an example:
   \code
   - \showdate "%lA %0d-%0m-%Y" 14-3-2015
   - \showdate "%a %d-%m-%y" 14-3-2015
   - \showdate "%m.%d%y" 14-3-2015
+  - \showdate "%lA %0d-%0m-%Y %H:%M:%S" 14-3-2015 03:04:15
+  - \showdate "%lA %0d-%0m-%Y %H:%M:%S" 14-3-2015 03:04
   \endcode
   In case `OUTPUT_LANGUAGE=english` this results in:
   - Saturday 14-03-2015
   - Sat 14-3-15
   - 3.1415
+  - Saturday 14-03-15 3:4:15
```

Comment:
This is a strange representation of 24h time, I would expect `03:04:15` (so just like the input format).

Reply:
It is but the choice of the user is `%H:%M:%S` but might have been better to use `%H:%0M:%S`

Reply:
I assume we want to follow e.g. strftime (https://man7.org/linux/man-pages/man3/strftime.3.html) so `%M` has range `00` to `59`, so with leading 0. There is no need for a version without leading 0 in my opinion.

Reply:
I tried to stay close to the definition as used e.g. for the month number in the `\showdate` command
>  | \%m | The month as a decimal number.
>  | \%0m | The month as a decimal number. If the result is a single digit, it is prefixed with 0.

The definition used in the strftime for month and hour are:

>        │%m  │                                             │
>        │    │        The month number, formatted with two │
>        │    │        digits (from `01' to `12'). [tm_mon] │

and

>        │%H  │                                             │
>        │    │        The hour (on a 24-hour clock),       │
>        │    │        formatted with two digits (from `00' │
>        │    │        to `23'). [tm_hour]

and with the date command and on https://cplusplus.com/reference/ctime/strftime/
> %m     month (01..12)
and
>  %H     hour (00..23)

So the initial implementation is also not following this definition.
With the current `\showdate` definitions it is possible have (when necessary) just e.g. a minutes or month value between 0 and 9 but also the values between 00 and 09.

Reply:
I think we should try to follow standards where possible. Maybe this one https://strftime.org/ that is used for Python can be followed. It has `%m` (standard) and the `%-m` extension for the non-zero padded variant.

Reply:
I think it would be possible to follow this "standard", I can adjust it in that way.
- Should I change it?

Reply:
I can also do it as part of the merge. Just let me know what you prefer

Reply:
Please do it as part of a merge.